### PR TITLE
[IN-45][Preprints] Truncate unicode strings correctly for Linkedin sharing.

### PR DESCRIPTION
**Pull request by adlius**
## Purpose

When a user clicks on the Linkedin sharing button on some preprints with Unicode encoded strings as titles or descriptions, the pop-up sharing page says "page not found". This is because Linkedin sharing works quite differently in that it gets sharing information from the query parameters of the sharing URL, which we construct on the front end side. Because javascript is not Unicode-aware, when we truncate Unicode strings according to specified length limit, it breaks the encoding and therefore causing errors for Linkedin when parsing the URL. 
This PR fix the problem by using an NPM package to do Unicode-aware string truncating.

## Summary of Changes

An NPM package `unicode-byte-truncate` is added as a dependency.
In `controllers/content/index.js`, `trunc` from the newly added dependency is used to truncate the title and description.

## Side Effects / Testing Notes

None.

## Ticket

https://openscience.atlassian.net/browse/IN-45

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,5 +96,8 @@
   "bugs": {
     "url": "https://github.com/CenterForOpenScience/ember-osf-preprints/issues"
   },
-  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme"
+  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme",
+  "dependencies": {
+    "unicode-byte-truncate": "^1.0.0"
```
Comment (jamescdavis) [inline package.json]: should go in devDependencies (see https://stackoverflow.com/questions/42159153/why-does-ember-installs-everything-as-devdependencies-instead-of-normal-dependen/42162627 for explanation of why this makes sense)

**Comment by adlius** [comment]
Some simple tests are added.

File: package.json [hunk 2]
```diff
@@ -91,10 +91,13 @@
...
-    "postcss": "5.1.0"
+    "postcss": "5.1.0",
+    "unicode-byte-truncate": "^1.0.0"
   },
   "bugs": {
     "url": "https://github.com/CenterForOpenScience/ember-osf-preprints/issues"
   },
-  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme"
+  "homepage": "https://github.com/CenterForOpenScience/ember-osf-preprints#readme",
+  "dependencies": {
```
Comment (baylee-d) [inline package.json]: If there isn't anything in `dependencies`, do we still need to include it in here?

**Comment by alexschiller** [comment]
Merge conflicts

**Pull request merged**